contrib/standalone-installer/assets/header.py

```diff
@@ -122,12 +122,43 @@ def install_flatpak (directory, verbose, uninstall):
     app = 'org.freedesktop.fwupd'
     common = '%s/.var/app/%s' % (os.getenv ('HOME'), app)
 
-    #check existing installed
-    cmd = ['flatpak', 'info', app]
     with open(os.devnull, 'w') as devnull:
+        if not verbose:
+            output = devnull
+        else:
+            output = None
+        #look for dependencies
+        dep = 'org.gnome.Platform/x86_64/3.28'
+        repo = 'flathub'
+        repo_url = 'https://flathub.org/repo/flathub.flatpakrepo'
```

Comment:
I think if we were installing this on the host computer they'd be an issue for Debian and Fedora, but I think this is fine as a helper script.

Reply:
Any other details?  I do envision that installer binaries built with this script should be usable on Debian and Fedora.  Flathub should work everywhere I'd think.

Reply:
At least feedback I've heard about folks trying this script so far is that on a vanilla install of something without any repos configured it errors out due to the lack of deps and repos to pull them from and it's not obvious what to do.

Reply:
Ya, I think it's fine. Lets see if Debian grumbles.